reference/gdscript.rst

```diff
@@ -799,9 +799,12 @@ special export syntax is provided.
     export(int) var number
 
     # Export can also take a resource type to use as a hint
-
+    
+    # Select Texture image
     export(Texture) var character_face
-
+    # Select a scene file from your project
+    export (PackedScene) var scene_file
+   
```

Comment:
Yes, to make it consistent with the rest of the code structure and commenting.
http://i.imgur.com/oSCvaNV.png